app/statistics/template.hbs

```diff
@@ -0,0 +1,137 @@
+<h1>Statistics</h1>
+
+{{fromDate}}
+<div>
+  {{#task-selection
+    history         = false
+    archived        = true
+    on-set-customer = (action 'setModelFilter' 'customer')
+    on-set-project  = (action 'setModelFilter' 'project')
+    on-set-task     = (action 'setModelFilter' 'task')
+    initial         = (hash
+      customer      = selectedCustomer
```

Comment:
Is this indent intended?

Reply:
Yeah. `initial` is an object created with the `hash` helper and `customer` is a property of `initial`. But `{{fromDate}}` on line 3 is a leftover I just discovered ^^

Reply:
ah yes, right ^^ Everything fine then